s2p/sift.py

```diff
@@ -287,13 +287,18 @@ def matches_on_rpc_roi(im1, im2, rpc1, rpc2, x, y, w, h,
         p2[:, 0] += x2
         p2[:, 1] += y2
         matches = np.hstack((p1, p2))
-        inliers = ransac.find_fundamental_matrix(matches, ntrials=1000,
-                                                 max_err=0.3)[0]
-        matches = matches[inliers]
         
         if len(matches) == 0 or matches.ndim != 2:
             print("WARNING: sift.matches_on_rpc_roi: found no matches.")
             return None
+        elif len(matches) > 10:
+            inliers = ransac.find_fundamental_matrix(matches, ntrials=1000,
+                                                 max_err=0.3)[0]
+            matches = matches[inliers]
+        else:
+            return matches
+        
```

Comment:
Maybe still add to return None if there are no matches after you filter using `inliers`